.env

```diff
@@ -29,7 +29,6 @@ DB_PASSWORD="dev-geouser-password"
 
 E2E_BROWSER=chrome
 
-E2E_REMOTE_SELENIUM_HUB_ENABLED=true
```

Comment:
Dockerin kautta näitä ajaessa tämän oltava true, joten ei sinänsä tarvii olla parami. Myöskään ne gradlen ui-tests-selenium-* profiilit ei tätä käytä joten poistettu hämäämästä